README.md

```diff
@@ -44,38 +44,38 @@ For questions and support please use the [Gitter chat room](https://gitter.im/ta
+|-----------------------------|-------------|--------------|-------|-------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| -cookie-domain              | string      |              | X     | Optional domain parameter for the cookie                                                                                                                    |
+| -cookie-expiry              | string      | session      | X     | Expiry duration for the cookie, e.g. 2h or 3h30m                                                                                                            |
+| -cookie-http-only           | boolean     | true         | X     | Set the cookie with the HTTP only flag                                                                                                                      |
+| -cookie-name                | string      | "jwt_token"  | X     | Name of the JWT cookie                                                                                                                                      |
+| -github                     | value       |              | X     | OAuth config in the form: client_id=..,client_secret=..[,scope=..,][redirect_uri=..]                                                                        |
+| -google                     | value       |              | X     | OAuth config in the form: client_id=..,client_secret=..,scope=..[redirect_uri=..]                                                                           |
+| -bitbucket                  | value       |              | X     | OAuth config in the form: client_id=..,client_secret=..,[,scope=..][redirect_uri=..]                                                                        |
+| -facebook                   | value       |              | X     | OAuth config in the form: client_id=..,client_secret=..,scope=email..[redirect_uri=..]                                                                      |
+| -host                       | string      | "localhost"  | -     | Host to listen on                                                                                                                                           |
+| -htpasswd                   | value       |              | X     | Htpasswd login backend opts: file=/path/to/pwdfile                                                                                                          |
+| -jwt-expiry                 | go duration | 24h          | X     | Expiry duration for the JWT token, e.g. 2h or 3h30m                                                                                                         |
+| -jwt-secret                 | string      | "random key" | X     | Secret used to sign the JWT token (if you don't use default, set related secret in http.jwt) (you can't use default and a custom one in the same caddyfile) |
```

Comment:
Better keep this string short here and mark it with a footnote where the different possibilities are explained in more detail. And be aware in the docu, that 50% of the loginsrv users use it standalone and should not care about caddy. They should not get confused by hints to http.jwt, which they properly do not know at all.

Reply:
Thanks for the note. I'll adjust doc differently.